### PR TITLE
deps: update dependency com.google.cloud:google-cloud-shared-dependencies to 2.10.2

### PR DESCRIPTION
**Pull request by losalex**
Update dependency com.google.cloud:google-cloud-shared-dependencies to 2.10.2 for 3.7.X LTS release branch

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -77,7 +77,7 @@
       <dependency>
         <groupId>com.google.cloud</groupId>
         <artifactId>google-cloud-shared-dependencies</artifactId>
-        <version>2.10.1</version>
+        <version>2.10.2</version>
```
Comment (suztomo) [inline pom.xml:80]:
```suggestion
        <version>2.10.3</version>
```

It's 2.10.3 that has gRPC 1.45.4.

**Review by suztomo — COMMENTED** [review]
It's 2.10.3 that has gRPC 1.45.4.

File: pom.xml [hunk 2]
```diff
@@ -77,7 +77,7 @@
       <dependency>
         <groupId>com.google.cloud</groupId>
         <artifactId>google-cloud-shared-dependencies</artifactId>
-        <version>2.10.1</version>
+        <version>2.10.3</version>
```
Comment (suztomo) [inline pom.xml:80]: Yes, this is correct one.

**Comment by suztomo** [comment]
Again, no need to merge this to the branch. Just to confirm the tests pass.

GraalVM tests are known to fail because the image uses the latest GraalVM version.

**Comment by losalex** [comment]
> Again, no need to merge this to the branch. Just to confirm the tests pass.
> 
> GraalVM tests are known to fail because the image uses the latest GraalVM version.

Sure, thank you. Seems some environment tests failed, but looking into logs seems it is related to the issue fixed before in min branch: [1261](https://github.com/googleapis/java-logging/pull/1261). All other tests are good, including E2E, so closing this PR as recommended

**Pull request closed without merging**